packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -87,38 +82,76 @@ class AbstractModal extends React.Component {
   constructor() {
     super();
     this.handleKeydown = this.handleKeydown.bind(this);
+    this.setAbstractModalRef = this.setAbstractModalRef.bind(this);
```

Comment:
Wanna use createRef for this guy? https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/608/commits/148cfbfb7fa7abcf5cd1ae14bf89bf2be7496b9d